.github/actions/setup-dependencies/action.yml

```diff
@@ -112,6 +112,51 @@ runs:
       uses: ./.github/actions/setup-matlab
       if: inputs.use_matlab == 'true'
 
+    #
+    # Android
+    # See https://developer.android.com/studio#command-tools for the latest version
+    #
+
+    # See https://github.com/marketplace/actions/android-emulator-runner
+    - name: Enable KVM group perms
+      shell: bash
+      run: |
+        echo 'KERNEL=="kvm", GROUP="kvm", MODE="0666", OPTIONS+="static_node=kvm"' | sudo tee /etc/udev/rules.d/99-kvm4all.rules
+        sudo udevadm control --reload-rules
+        sudo udevadm trigger --name-match=kvm
+      if: matrix.config == 'android'
+
+    - name: Install Android SDK
```

Comment:
Can't we use the ones included with the Ubuntu images?

See https://github.com/actions/runner-images/blob/main/images/ubuntu/Ubuntu2404-Readme.md#android

Reply:
are these env variables already set?

Reply:
Seems likely.